torchqdynamics/mesolve/adaptive.py

```diff
@@ -1,14 +1,13 @@
 from torch import Tensor
 
 from ..odeint import ForwardQSolver
-from ..solver_options import SolverOption
 from ..solver_utils import kraus_map
 from ..tensor_types import TDOperator
 
 
 class MEAdaptive(ForwardQSolver):
-    def __init__(self, options: SolverOption, H: TDOperator, jump_ops: Tensor):
-        super().__init__(options)
+    def __init__(self, *args, H: TDOperator, jump_ops: Tensor):
+        super().__init__(*args)
```

Comment:
Here and in all other ForwardQSolver children classes, I think it would be nice to have a more meaningful name than `*args` which implies any argument. Maybe `*qsolver_args` ? This way we know it is what is passed to the parent class.

Reply:
It's very canonical to name positional arguments `*args` and keyword-only arguments `**kwargs`, and even more canonical in inheritance-like schemes. From what I read, it's not recommended to pick different names, because everyone in the Python community uses this, so I would argue to keep it like that.